### DISCUSSION TITLE
Alguien podria ayudarme a entender como podria resolver el problema en el caso que tuviera un nodos con 3 ventanas de tiempo?

### DISCUSSION THREAD

**Discussion by rocio734** [google/or-tools#3372] [Routing (and legacy CP) questions]
Si en el input le envio 3 ventanas de tiempo a un nodo falla debido al calculo que hace para calcular los intervalos de tiempo, como deberia hacer para resolverlo?

**Comment by lperron** [comment]
my spanish is not good enough

**Comment by lesshaste** [comment]
"If in the input I send 3 time windows to a node it fails due to the calculation it does to calculate the time intervals, how should I do to solve it?"

**Comment by Mizux** [comment]
What did you try so far ?

supposing you have three TW: `[t0, t1]`, `[t2, t3]`, `[t4, t5]`.
then:
1) create your dimension with at least `t5` as max capacity
2) remove unneeded intervasl using `RemoveInterval()` aka `[0, t0]`, `[t1, t2]`, `[t3, t4]`
see: https://github.com/google/or-tools/blob/2cb85b4eead4c38e1c54b48044f92087cf165bce/ortools/constraint_solver/constraint_solver.h#L4056-L4058
